### ISSUE TITLE
A11y_AzureDataStudioNew_Database_ScreenReader: ORCA Screen reader is not announcing selected or non selected state of list item from the dropdown.

### ISSUE THREAD

**Issue by PoojaNamde** [microsoft/azuredatastudio#23182]
"[Check out Accessibility Insights!] - Identify accessibility bugs before check-in and make bug fixing faster and easier."

### #GitHub Tags:
#A11yMAS; #A11ySev2; #A11yTCS; #DesktopApp; #SH-AzureDataStudio-Linux-Jul2021; #Azure Data Studio; #WCAG1.3.1; #Linux; #Element:EditField; #ScreenReader; #ORCA;

### Actual Result:​
ORCA Screen reader is not announcing selected or non selected state of list item from the "Charset" and "Collation" dropdown.

### Expected Result:​
ORCA Screen reader should announce selected or non selected state of list item from the "Charset" and "Collation" dropdown.
For example we have selected "utf8mb4" list item then screen reader should announce it as "Charset utf8mb4 selected".

### Environment Details:​
Application Name: MySQL extension for Azure Data Studio
Application Version: 1.42.0-insider
Operating System: Ubuntu 20.04.2 LTS
GNOME Version: 3.36.8
ORCA: 42.2-1ubuntu1
Distributor ID: Ubuntu
Description: Ubuntu 22.10
Release: 22.10
Codename: kinetic

### Repro Steps:

1. Launch Azure Data Studio application.
2. TAB to "Servers" and select any server from the list and hit ENTER on 'Connect' button.
3. TAB to connected server and right click on it and TAB to "Manage" and hit ENTER key.
4. TAB to "New Dashboard" and hit ENTER key.
5. "New Dashboard" screen gets open. Start the ORCA screen reader.
6. TAB to "Charset" and "Collation" dropdown and hit ENTER key.
7. Observe that ORCA Screen reader is not announcing selected or non selected state of list item from the dropdown.

### User Impact:
Screen reader users will get confused about the announcement as they might have selected any list item but it is not announcing as selected.

**"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility or the Divisional Driver."**

### Attachment:

[228767830-4a1c5bd9-6665-4fd7-847f-4d37e54e5f7f.webm](https://github.com/microsoft/azuredatastudio/assets/90901519/5d435305-6567-4dd3-94ef-517b52b64459)

**Comment by PoojaNamde** [comment]
Primary bug: https://github.com/microsoft/azuredatastudio-mysql/issues/172

**Comment by alanrenmsft** [comment]
@PoojaNamde I believe this is an issue with ORCA, we are just using a standard select element here, you can reproduce the same issue with this example here: https://www.w3schools.com/tags/tryit.asp?filename=tryhtml_select

**Comment by PoojaNamde** [comment]
#Closed; #Regressed:06-06-23;

Closing this bug as per the discussion with SME.
